2022/covid-19.html

```diff
@@ -0,0 +1,121 @@
+                regulations</a> and guidelines, and when deemed necessary, take
+                additional measures at our discretion. As regulations and best practices around COVID management for
+                events evolve constantly, we may change our approach up to the last moment.
+
+                The latest version of this policy is available at <a href="https://www.jbcnconf.com/2022/covid-19.html">COVID-19
+                Policy</a>.
+
+                If you have any questions or concerns about our COVID-19 policy, please do not hesitate to contact us
+                via <a href="mailto:info@barcelonajug.org">info@barcelonajug.org</a>.
+
+                <h3>What we expect from you</h3>
+                This policy applies to all participants, including visitors, speakers, organisers, vendors, sponsors,
+                and staff.
+
+                During the event, as well during your interactions with other participants outside the event, we expect
+                you to adhere to both <a
+                href="https://web.gencat.cat/en/activem/restriccions-territorials/catalunya/#bloc7">local
+                governmental</a> measures and our additional measures.
+
+                Our measures may include:
+                <ul>
+                  <li>Providing proof of vaccination, or a recent (<24h) negative test, upon entry of the venue</li>
+                  <li>Mandatory use of face-masks</li>
+                  <li>Testing attendees upon entry</li>
+                  <li>Limiting capacity to certain sessions</li>
+                  <li>This list is non-exhaustive.</li>
+                </ul>
+
+                If a participant doesn't comply to the policy, the event organisers may take any action they deem
+                appropriate, including warning the offender or removing them from the event with no right to a refund.
+
+                <h3>What you can expect from us</h3>
+                Together with the venue and our suppliers, we will take appropriate measures to create safe conditions
+                for everyone, based on regulations, best practices, and guidelines.
+
+                Our measures may include:
+                <ul>
+                  <li>Creating fresh air flows</li>
+                  <li>Providing masks and disinfectant dispensers</li>
+                  <li>Adapting catering for safety</li>
+                  <li>Limiting capacity for certain activities</li>
+                  <li>Air quality control monitoring</li>
+                  <li>This list is non-exhaustive.</li>
+                </ul>
+
+                <h3>What if you can’t attend anymore?</h3>
+                Our general <a href="https://www.jbcnconf.com/2022/conditions.html">Terms and Conditions</a> apply for
+                cancellations by the participant.
+
+                However, under the following conditions, we will refund the full cost of the ticket when:
```

Comment:
I would consider deleting that whole section, as we cannot accept refunds in these cases + also we can't say different things as we have in our Terms And Conditions page.
 
The other is to say that we invite of course someone else to make use of that ticket.

I think I'm more up to suggesting removing this section and the next one... because they are not directly related to COVID-19. They are in fact related to the general Terms and Conditions of our event... but up to you if you prefer to change it...